src/openzeppelin/token/erc20/presets/ERC20Upgradeable.cairo

```diff
@@ -1,168 +1,121 @@
-# SPDX-License-Identifier: MIT
-# OpenZeppelin Contracts for Cairo v0.3.2 (token/erc20/presets/ERC20Upgradeable.cairo)
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts for Cairo v0.3.2 (token/erc20/presets/ERC20Upgradeable.cairo)
 
 %lang starknet
 %builtins pedersen range_check
 
 from starkware.cairo.common.cairo_builtins import HashBuiltin
-from starkware.cairo.common.bool import TRUE
 from starkware.cairo.common.uint256 import Uint256
 
 from openzeppelin.token.erc20.library import ERC20
 from openzeppelin.upgrades.library import Proxy
 
-#
-# Initializer
-#
+//
+// Initializer
+//
 
 @external
-func initializer{
-        syscall_ptr: felt*,
-        pedersen_ptr: HashBuiltin*,
-        range_check_ptr
-    }(
-        name: felt,
-        symbol: felt,
-        decimals: felt,
-        initial_supply: Uint256,
-        recipient: felt,
-        proxy_admin: felt
-    ):
-    ERC20.initializer(name, symbol, decimals)
-    ERC20._mint(recipient, initial_supply)
-    Proxy.initializer(proxy_admin)
-    return ()
-end
+func initializer{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+    name: felt,
+    symbol: felt,
+    decimals: felt,
+    initial_supply: Uint256,
+    recipient: felt,
+    proxy_admin: felt,
+) {
+    ERC20.initializer(name, symbol, decimals);
+    ERC20._mint(recipient, initial_supply);
+    Proxy.initializer(proxy_admin);
+    return ();
+}
 
 @external
-func upgrade{
-        syscall_ptr: felt*,
-        pedersen_ptr: HashBuiltin*,
-        range_check_ptr
-    }(new_implementation: felt):
-    Proxy.assert_only_admin()
-    Proxy._set_implementation_hash(new_implementation)
-    return ()
-end
-
-#
-# Getters
-#
+func upgrade{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
```

Comment:
Just a reminder of the **upgradeAndCall** suggestion to keep the atomicity if the new implementation needs initialization.